examples/odata-cli/src/activities.ts

```diff
@@ -180,3 +183,93 @@ export async function testDeployUndeployDTA(
         logger.error(`Error: ${error.message}`);
     }
 }
+
+/**
+ * Parse an XML document for ATO (Adaptation Transport Organizer) settings.
+ *
+ * @param xml xml document containing ATO settings
+ * @returns parsed ATO settings
+ */
+function _parseResponse<T>(xml: string): T {
```

Comment:
No reference found within this file that uses this function.

Reply:
If it is not needed, then we could also get rid of the dependency `fast-xml-parser` correct?

Reply:
Thanks, both are removed now

Reply:
@docirl  I still see fast-xml-parser referenced in `examples/odata-cli/package.json`.

Reply:
This is not critical since it is in the example only.